src/UI/Badge.elm

```diff
@@ -1,55 +1,58 @@
-module UI.Badge exposing (dark, error, light, primary, success, warning)
+module UI.Badge exposing (Badge, badge, toEl)
 
-import Element exposing (..)
+import Element exposing (Element, px, shrink)
 import Element.Background as Background
 import Element.Border as Border
 import Element.Font as Font
-import UI.Theme as Theme
+import UI.Internal.Palette as Palette
+import UI.Internal.Primitives as Primitives
+import UI.RenderConfig exposing (RenderConfig)
+import UI.Text as Text
 
 
-type alias ColorScheme =
-    { font : Color
-    , background : Color
-    }
+type Badge
+    = Badge Properties Options
 
 
-light : String -> Element msg
-light =
-    view { font = Theme.black, background = Theme.gray3 }
+type alias Properties =
+    { content : String }
 
 
-primary : String -> Element msg
-primary =
-    view { font = Theme.white, background = Theme.primary }
+type alias Options =
+    { tone : BadgeTone }
 
 
-dark : String -> Element msg
-dark =
-    view { font = Theme.white, background = Theme.black }
+type BadgeTone
+    = ToneDefault
 
 
-success : String -> Element msg
-success =
-    view { font = Theme.black, background = Theme.success }
+badge : String -> Badge
+badge content =
+    Badge { content = content } defaultOptions
 
 
-warning : String -> Element msg
-warning =
-    view { font = Theme.black, background = Theme.warning }
 
+-- Render
 
-error : String -> Element msg
-error =
-    view { font = Theme.black, background = Theme.error }
 
+toEl : RenderConfig -> Badge -> Element msg
+toEl cfg (Badge { content } { tone }) =
+    Text.overline content
+        |> Text.toEl cfg
+        |> Element.el
+            [ Element.width shrink
+            , Font.center
+            , Background.color Palette.gray.lighter
+            , Element.paddingEach { top = 4, bottom = 4, left = 5, right = 3 }
```

Comment:
left and right padding should normally have the same number, I checked the sketch inspector and it had 5, so I think you could have `paddingXY 5 4`

Reply:
Letter-spacing is moving text 1px to the left on both Chrome and Firefox for me!